src/App.vue

```diff
@@ -2,6 +2,10 @@
   <div>
     <the-header></the-header>
     <main>
+      <carousel-list
+        v-if="productList.length"
+        :carousel-list="carouselListData"
+      ></carousel-list>
```

Comment:
```suggestion
      />
```
Nitpick, but I believe it is in the recommended Vue style guide to self-close components without body. 😉